### PR TITLE
Fix after install URL and new photo since AcmeDemoBundle is not part …

### PR DESCRIPTION
**Pull request by smatejic**
Since removal of AcmeDemoBundle doc's should have new image and app/example URL

### CODE REVIEW

**Comment by WouterJ** [comment]
:+1:

**Comment by weaverryan** [comment]
Thank you @smatejic - perfect PR!

**Comment by harikt** [comment]
Thank you @smatejic . 

I am working with a broken internet, sorry for not able to make a PR my own.

**Comment by weaverryan** [comment]
@harikt team work is even better! ;)

File: book/installation.rst [hunk 1]
```diff
@@ -169,7 +169,8 @@ browsing the project directory and executing this command:
     $ cd my_project_name/
     $ php app/console server:run
 
-Then, open your browser and access the ``http://localhost:8000`` URL to see the
+Then, open your browser and access the ``http://localhost:8000/app/example``
+URL to see the
 Welcome page of Symfony:
```
Comment (stof) [inline book/installation.rst]: this is not the welcome page anymore though

**Comment by harikt** [comment]
By the way is there a nice example bundle from where we can learn the best practices / usage ? I know there is a link to the best practices, but may be a good bundle for the current demo is no more.

**Comment by stof** [comment]
See https://github.com/symfony/symfony-demo

**Comment by smatejic** [comment]
Well

Demo is now part of Symphony Demo Application

https://github.com/symfony/symfony-demo

http://symfony.com/blog/introducing-the-symfony-demo-application

**Comment by harikt** [comment]
Thank you @smatejic , @stof .

When reading the chapters I have not came across a link to demo application or may I missed. It will be nice to refer the demo application in the starting, so people can get familiar with things. What do you guys think ?

**Comment by weaverryan** [comment]
@harikt you're totally right. We've got a PR open for it :) #5355

**Pull request merged**